aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -1,91 +1,85 @@
-# Glue setup
-import sys
-from awsglue.transforms import *
-from awsglue.utils import getResolvedOptions
-from pyspark.context import SparkContext
+import re
+import time
+
+import boto3
+import numpy as np
+import pandas as pd
+from assesspy import boot_ci, cod
+from assesspy import cod_ci as cod_boot
+from assesspy import cod_met, detect_chasing, mki, mki_met, prb, prb_met, prd
+from assesspy import prd_ci as prd_boot
+from assesspy import prd_met
```

Comment:
**[Nitpick, non-blocking]** We could collapse this all into one import

```suggestion
from assesspy import (
    boot_ci,
    cod,
    cod_ci as cod_boot,
    cod_met,
    detect_chasing,
    mki,
    mki_met,
    prb,
    prb_met,
    prd,
    prd_ci as prd_boot,
    prd_met
)
```

Reply:
It turns out that `isort` really hates the more pythonic syntax, so sticking with Damon's original imports.